com.leohanney.Volaris.yml

```diff
@@ -0,0 +1,38 @@
+id: com.leohanney.Volaris
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+          
+command: volaris-gui
+rename-icon: com.leohanney.Volaris
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+
+          
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/volar-is/volaris-gui/releases/download/app-v0.1.0/Volaris_0.1.0_amd64.deb
+        sha256: c55febeee6cbde904b52119e86606dd08eb474aa8fc53be16843841334cdcc3c
+        only-arches: [x86_64]
+      - type: file
+        url: https://raw.githubusercontent.com/volar-is/volaris-gui/main/flatpak.xml
```

Comment:
this needs to be from a commit not `main`.

Reply:
why isn't it in the .deb? it's clearly misnamed.